src/components/Dashboard/FirstSteps/ActionBox/index.js

```diff
@@ -6,15 +6,20 @@ import {
   PENDING_STATUS,
   WARNING_STATUS,
 } from '../reducers/firstStepsReducer';
+import { useIntl } from 'react-intl';
+import { FormattedMessageMarkdown } from '../../../../i18n/FormattedMessageMarkdown';
 
 export const ActionBox = ({ status, title, description, textStep }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
   return (
     <div className={`dp-step ${INFO_BY_STATE[status].classNames}`} role="alert" aria-label="step">
       <h4>
         <span className="dp-iconstep">{status === PENDING_STATUS ? textStep : ''}</span>
-        {title}
+        {_(title)}
```

Comment:
Maybe we should make clear that it is not the literal text but an Id.

```suggestion
        {_(titleId)}
```

Reply:
Applied!